SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -206,4 +206,6 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	TWeakObjectPtr<ASpatialDebugger> SpatialDebugger;
 
 	TOptional<FMonitoredProcess> InspectorProcess = {};
+
+	uint16_t PreviousRuntimeGRPCPort;
```

Comment:
Maybe worth calling this `CachedRuntimeGRPCPort`, little more fancy

Reply:
Will do!